### DISCUSSION TITLE
Can we send silent push notification using the ?

### DISCUSSION THREAD

**Discussion by sarayoodharan** [relayrides/pushy#1034] [Q&A]
Can we send silent push notification using the Pushy? 

We have requirement where we have to send a silent push with "content-available" : 1 in the payload.
So, I am wondering that can be achieved.

Thanks,
Sarayoodharan.K

**Comment by jchambers** [comment]
Yes. Please see [`ApnsPayloadBuilder#setContentAvailable(boolean)`](https://pushy-apns.org/apidocs/latest/com/eatthepath/pushy/apns/util/ApnsPayloadBuilder.html#setContentAvailable(boolean)).
